docs/nidcpower/class.rst

```diff
@@ -1298,45 +1298,6 @@ get_ext_cal_recommended_interval
 
 
 
-get_lcr_compensation_data
```

Comment:
CHANGELOG.md needs corresponding change.

Reply:
I've pushed a commit that removes `get_lcr_compensation_data` from CHANGELOG.md.